.travis.yml

```diff
@@ -64,3 +69,15 @@ script:
   - make check
   - make deb
 
+# Deploy debian packages to bintray
+deploy:
+    provider: script
+    script: beaver bintray upload -N pkg/deb/*.deb
+    skip_cleanup: true
+    on:
+        tags: true
+        condition: >
+            "$TRAVIS_OS_NAME" = "linux" -a
+            "$DIST_PACKAGE_TARGET" = "DEB" -a
```

Comment:
1. I would assume that **DIST_PACKAGE_TARGET** should be declared under the `env` to be used in the comparison. 
2. With the current .travis settings, the conditions `$TRAVIS_OS_NAME == linux` and `"$CC" = "gcc"` would be sufficient to distinguish the build. I.e **DIST_PACKAGE_TARGET** isn't needed until the changes added in `v5.x.x`